components/dialog/test/dialog-confirm.test.js

```diff
@@ -19,8 +20,8 @@ describe('d2l-dialog-confirm', () => {
 		it('should focus on first non-primary button', async() => {
 			const el = await fixture(html`
 				<d2l-dialog-confirm opened>
-					<button slot="footer" primary>Yes</button>
-					<button slot="footer">No</button>
+					<d2l-button slot="footer" primary>Yes</d2l-button>
```

Comment:
This test is skipped anyways, but the analyzer wasn't happen with putting `primary` on a native button so I've just switched them to `d2l-button`.